src/Schema/Events/Consignments.ts

```diff
@@ -80,7 +80,33 @@ export interface UploadPhotosCompleted {
   action: ActionType.uploadPhotosCompleted
   context_module: ContextModule.uploadPhotos
   context_owner_type: OwnerType.consignmentFlow
-  submission_id: string
+  submission_id?: string
+  user_email: string
+  user_id?: string
```

Comment:
**note:** I would have set this to be `string | undefined` instead. I always prefer explicit rather than implicit in such contexts

Reply:
Noted.

Reply:
@MounirDhahri Hahaha. You had merged before I could update it. No problem.

Reply:
oh sorry Kizito, I just wanted to leave a note but it's definitely not blocking.